src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2452,8 +2470,22 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
 
             if (offset_counter(p) == dt) then
                t1 = 1.0_r8 / dt
-               leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
+               !leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
                frootc_to_litter(p) = t1 * frootc(p) + cpool_to_frootc(p)
+               
+               ! Y. Cheng, cut 70% AGB for switchgrass and Miscanthus and move to grainc_to_food
+               if (ivt(p)==nmiscanthus .or. ivt(p)==nirrig_miscanthus .or. ivt(p)==nswitchgrass .or. ivt(p)==nirrig_switchgrass) then
+               	   leafc_to_litter(p)  = t1 * leafc(p)*0.3  + cpool_to_leafc(p)
+               	   grainc_to_food(p) =grainc_to_food(p)+ t1 * leafc(p) * 0.7 
```

Comment:
Would you also want to make a change where currently the grain pool is being added to grainc_to_food (line 2502), instead putting that in the new biofuel flux?

Reply:
I agree that putting a product where leaf biomass is added into the grainc_to_food variable is confusing. It's probably best to keep the grainc_to_food variable similar to data available for evaluation, and UNFAO-STAT does not include biofuels in the global grain yield product. As per @wwieder and @billsacks suggestions, is it possible to change this to a new biofuelc flux, where both the grain pool and 70% of the leaf biomass are put into the biofuel flux? It will also then be easier to keep track of the total biofuel production independently of grain products. The biofuelc flux can follow the same pathway as grainc_to_food, where it ends up in the 1-year product pool.

Reply:
I also wonder: Would it make sense to introduce another crop-specific parameter for this, specifying the fraction of crop grain that goes to biomass vs. food for each crop? My main thinking is that I'd like to keep logicals on crop type out of the code as much as possible, but I also wonder if this would be a useful flexibility to have. I don't feel strongly about this, but am throwing it out as a suggestion in case others think it could be good to do.

Reply:
So @billsacks you are saying that the fraction of biomass that goes to biofuel would be a parameter that is an array of PFT. And likewise the fraction of biomass that goes to food would be a parameter that is an array of PFT. And then this statement wouldn't need to be encapsulated in an if statement but done for every crop type. That is better. If statements tend to be buggy and cause confusion.

Reply:
Yes, something like that. We could discuss details of what the parameters should be.

Reply:
Yes, this is what I was trying to get across in my earlier comment related to residue harvest. It would be great to have crop-specific parameters for this. I think the challenge will be linking the parameters to the appropriate fluxes (e.g., the grainc_to_food vs litter vs cropprod_1yr), which will also likely vary by crop type.